src/ocean/text/Arguments.d

```diff
@@ -611,6 +611,7 @@ import ocean.io.stream.Format : FormatOutput;
 import ocean.math.Math;
 import ocean.text.Util;
 import ocean.text.convert.Formatter;
+static import ocean.text.convert.Layout_tango; // TODO: Remove in v4.0.0
```

Comment:
Create an issue for this?

Reply:
https://github.com/sociomantic-tsunami/ocean/issues/206

Reply:
You can also include the keyword "deprecated" in the comment, so we'll find it when searching for old crap to remove.